teleprox/tests/test_spawner.py

```diff
@@ -29,18 +33,26 @@ def test_serverless_client():
     assert os.getpid() != ros.getpid()
 
     class CustomType:
-        def __init__(self):
```

Comment:
Why remove this?

Reply:
I believe an empty class is all that's needed to test the behavior under scrutiny here.